.kokoro/healthcare/datasets.cfg

```diff
@@ -0,0 +1,13 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Set the folder in which the tests are run
+env_vars: {
+    key: "PROJECT"
+    value: "healthcare/datasets"
+}
+
+# Tell the trampoline which build file to use.
+env_vars: {
```

Comment:
can we move this section to common.cfg because it's the same for all the healthcare samples?

Reply:
Done.